### DISCUSSION TITLE
Import HTML page with javascript and css

### DISCUSSION THREAD

**Discussion by beto580** [requarks/wiki#2999] [Help / Questions] [answered]
OS: Ubuntu 18.04 and Docker
Wiki.js version: 2.5.170
Database engine: PostgreSQL 9.6

Hello, first of all, let me congratulate you for all the amazing work done with wiki.js

I work at a company in Argentina named N5 Now. We are implementing our documentation using Wiki.js.

I successfully installed wiki.js on our server but I wanted to try to import an HTML page with some existing javascript and CSS. 
I tried adding a new page and selecting Code (Raw HTML) then added some javascript and some CSS in the corresponding tabs and some HTML referencing the scripts. I wasn't able to make it work.

Could you give me some pointers?

Thank you.

**Comment by NGPixel** [comment, marked as answer]
You need to insert the JS and CSS under the Page Properties > Scripts / CSS dialog while editing the page. You cannot add JS or CSS directly into the page content.

**Comment by beto580** [comment]
Thanks nicolas!

I tried to do that but I was having problems with javascript functions.
Defining one in the script tab and calling it on the page content. Could
you provide an example of this or an alternative method?

**Reply by cloudliu01** [reply]
I am experiencing the same issue. Tried to add a sortable table into a WIKIJS page (HTML Raw) but never worked. Put all css, scripts and html on the page (not to the style or javascript tab), the sortable table worked right after the main page, but part of the table was blocked by the navagation widget on the left side so not very useful.

Could someone help check how to get it working?

The test table's code is here: https://www.jquery-az.com/boots/demo.php?ex=19.0_7

And the problem is illustrated in the screenshot below:
![Screenshot 2022-02-19 at 20 00 28](https://user-images.githubusercontent.com/15914527/154817318-f3d8ad9c-65e0-47f5-b6b7-c21bee671211.png)
